kimchi/src/circuits/polynomials/chacha.rs

```diff
@@ -518,10 +567,12 @@ mod tests {
             h.insert(Column::Index(GateType::Generic));
             h
         };
-        let mut expr = constraint_chacha0(0..5);
-        expr += constraint_chacha1(0..5);
-        expr += constraint_chacha2(0..5);
-        expr += constraint_chacha_final(0..9);
+        let mut alphas = Alphas::<F>::default();
+        alphas.register(ArgumentType::Gate(GateType::ChaChaFinal), 9);
```

Comment:
perhaps a counter for this 9? I bet this was suggested before, not so important I guess